src/components/head.tsx

```diff
@@ -2,12 +2,20 @@ type MetaData = {
   title: string;
   description: string;
   icon: string;
+  ogImage?: string;
+  ogUrl?: string;
+  twitterCard?: string;
+  twitterSite?: string;
 };
 
 const DEFAULT_META_DATA: MetaData = {
   title: "React Tokyo",
   description: "A React community in Tokyo",
```

Comment:
[IMO] 説明文が分かりづらいのでサイトに掲載してる文言そのまま使ってみても良いかと思いました！

```suggestion
  description: "東京を中心に、オンラインとオフラインの両方で活動するReactコミュニティです。オンラインでは、Discordを通じた情報交換や交流の場を提供し、オフラインではReactに関するイベントを開催します。",
```

※画像はプレビュー環境なので映らない
<img width="540" alt="image" src="https://github.com/user-attachments/assets/5bff3424-dff5-473d-bc0f-6ac0838dd751" />